codebuild/bin/grep_simple_mistakes.sh

```diff
@@ -39,7 +39,7 @@ declare -A KNOWN_MEMCMP_USAGE
 KNOWN_MEMCMP_USAGE["$PWD/crypto/s2n_rsa.c"]=1
 KNOWN_MEMCMP_USAGE["$PWD/tls/s2n_early_data.c"]=1
 KNOWN_MEMCMP_USAGE["$PWD/tls/s2n_kem.c"]=1
-KNOWN_MEMCMP_USAGE["$PWD/tls/s2n_cipher_suites.c"]=3
+KNOWN_MEMCMP_USAGE["$PWD/tls/s2n_cipher_suites.c"]=4
```

Comment:
Is there a reason we can't use s2n_constant_time_equals() ? see #3062

Reply:
Yeah we definitely want this number to go down; not up

Reply:
No reason, I just reused the existing function that used `memcmp`.